src/main/res/values/strings.xml

```diff
@@ -448,6 +449,7 @@
     <string name="send_message_to">Send Message to…</string>
     <string name="enable_realtime">Real-Time Apps</string>
     <string name="enable_realtime_explain">Enable real-time connections for apps shared in chats. If enabled, chat partners may be able to discover your IP address when you start an app.</string>
+    <string name="realtime_disabled_warning">\"Real-time\" feature is disabled, this app may not work properly. Do you want to enable it? Chat partners may be able to discover your IP address</string>
```

Comment:
call the feature here as it is called in the settings. and do not forget the final, serious dot.

```suggestion
    <string name="realtime_disabled_warning">\"Real-Time Apps\" feature is disabled, this app may not work properly. Do you want to enable it? Chat partners may be able to discover your IP address.</string>
```